.github/workflows/ci.yaml

```diff
@@ -160,9 +160,6 @@ jobs:
           config: ./.typos.toml
 
       - uses: taiki-e/install-action@cargo-machete
-      - name: Check unused dependencies
-        run: cargo machete --skip-target-dir
```

Comment:
I have some rough guesses but is there any reason we want to do away with `cargo-machete`

Reply:
I removed him from here, because now it's inside Makefile as `lint-machete`, so when you do `make lint` we are already linting machete also.